day01/toArray.js

```diff
@@ -0,0 +1,5 @@
+module.exports = input =>
```

Comment:
I like that you've pulled this out into its own file! Another step you may want to consider in future things is to contain helper functions like this into a `helpers` directory or `utils` directory, or something of the like. I think that's sort of what you've already established with `lib`, so maybe this could live there?

Along that line of thinking, you could also consider exporting this as a [named export](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export) so that you could use it elsewhere and it'd have a familiar name to you and other folks who might encounter this bit of code.

I found [this article](https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/) to be an interesting look at using default vs. named exports.

Reply:
Thanks for the article suggestion @pfulton I'll keep this in mind moving forward.